### PR TITLE
I have added business logic in service layer

### PR DESCRIPTION
**Pull request by Yogesh-31B**
Please Check my code carefully . It has lot if trozens.

### CODE REVIEW

File: src/main/java/com/yas/controller/CustomerController.java [hunk 1]
```diff
@@ -14,32 +14,27 @@
...
-	
+
 	@GetMapping("/get-customer")
-	public List<Customer> getCustomer(){
-		return cr.findAll();
+	public List<Customer> getCustomer() {
+		return cs.getCustomer();
 	}
-	 
+
```
Comment (ankita-gurbhele) [inline src/main/java/com/yas/controller/CustomerController.java:34]: plz change the annotation

File: src/main/java/com/yas/service/CustomerService.java [hunk 2]
```diff
@@ -1,27 +1,46 @@
...
+	@Autowired
+	ServicesRepository sr;
+
 	public Customer getCustomer(Customer customer) {
-		Customer custom = cr.findByAll(customer.getFirstName(), 
-									   customer.getLastName(),
-									   customer.getAddress(),
-									   customer.getMobNo(),
-									   customer.getEmail());
+		Customer custom = cr.findByAll(customer.getFirstName(), customer.getLastName(), customer.getAddress(),
```
Comment (ankita-gurbhele) [inline src/main/java/com/yas/service/CustomerService.java:24]: check this logic again

**Review by ankita-gurbhele — CHANGES_REQUESTED** [review]

File: src/main/java/com/yas/controller/CustomerController.java (hunk 1 above)
Comment (Yogesh-31B) [reply src/main/java/com/yas/controller/CustomerController.java:34]: I have added the annotation please check it .

File: src/main/java/com/yas/service/CustomerService.java (hunk 2 above)
Comment (Yogesh-31B) [reply src/main/java/com/yas/service/CustomerService.java:24]: I have done unit testing please review it.

**Review by Yogesh-31B — COMMENTED** [review]
Okay thanks for your patience . Please check it again

**Review by ankita-gurbhele — APPROVED** [review]

**Comment by ankita-gurbhele** [comment]
Well done Yogesh!

**Pull request merged**